File: Data Structures & Algorithms/non-overlapping-intervals/submission-11.js
class Solution {
    /**
     * @param {number[][]} intervals
     * @return {number}
     */
    eraseOverlapIntervals(intervals) {
        intervals.sort((a,b) => a[1] - b[1]);
        let inital = intervals[0]
        let res = 0
        for(let i = 1; i < intervals.length; i++){
            if(intervals[i][0] < inital[1]){
                res++
            }else{
                inital[1] = intervals[i][1]
            }
        }
        return res
    }
}

File: Data Structures & Algorithms/merge-intervals/submission-11.py
class Solution:
    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
        intervals.sort()
        ret = [intervals[0]]

        for x,y in intervals[1:]:
            if(ret[-1][1] < x):
                ret.append([x,y])
            else:
                ret[-1][0] = min(x, ret[-1][0])
                ret[-1][1] = max(y, ret[-1][1])
        
        return ret